IPython/core/display.py

```diff
@@ -27,7 +27,8 @@
 'display_javascript', 'display_pdf', 'DisplayObject', 'TextDisplayObject',
 'Pretty', 'HTML', 'Markdown', 'Math', 'Latex', 'SVG', 'JSON', 'GeoJSON', 'Javascript',
 'Image', 'clear_output', 'set_matplotlib_formats', 'set_matplotlib_close',
-'publish_display_data', 'update_display', 'DisplayHandle', 'Video']
+'publish_display_data', 'update_display', 'DisplayHandle', 'Video', 'get_repr_mimebundle',
+'RecursiveObject', 'ReprGetter']
```

Comment:
RecursiveObject and ReprGetter should probably *not* be added to `__all__`, since they are private, internal implementation details.